sql/src/test/java/io/crate/integrationtests/IngestRulesIntegrationTest.java

```diff
@@ -88,4 +76,17 @@ public void testDropMissingRuleFails() {
         expectedException.expectMessage("SQLParseException: Ingest rule somerule doesn't exist");
         execute("drop ingest rule somerule");
     }
+
+    @Test
+    public void testDropTargetTableRemovesAssociatedRules() {
+        execute("drop table t1");
+
+        // TODO: replace this with a query on information schema when available
```

Comment:
isn't the information schema table already implemented? can we solve this todo right now?